r/R/worker_helpers.R

```diff
@@ -199,3 +197,23 @@ handlePagination <- function(gene_results, offset, limit, order_by, order_decrea
   gene_results <- na.omit(gene_results[(offset):(offset + limit), ])
   return(list(gene_results = gene_results, full_count = full_count))
 }
+
+insert_set_child_through_api <- function(new_cell_set, api_url, experiment_id, cell_set_key, auth_JWT) {
```

Comment:
Can we do this patch from the Python worker instead? We have this code in the Python worker which does that: https://github.com/biomage-ltd/worker/blob/7a0227e82ebcc7b3ad96290d3224e9416a6fb498/python/src/worker/tasks/cluster_cells.py#L54

I think the Python worker is more well equipped with communicating with the API. It can also access the whole request body.